programs/tensorflowDetection2D/TensorflowDetector.cpp

```diff
@@ -0,0 +1,227 @@
+
+// Librerias
+
+#include <math.h>
+#include <fstream>
+#include <utility>
+#include <vector>
+#include <iostream>
+#include <regex>
```

Comment:
We need C++11, then. In your CMakeLists.txt, instruct CMake to request a C++11-capable compiler for this target:

```cmake
set_target_properties(myTarget PROPERTIES
    CXX_STANDARD 11
    CXX_STANDARD_REQUIRED YES
    CXX_EXTENSIONS NO
)
```

There might be better options, though. Ref: https://crascit.com/2015/03/28/enabling-cxx11-in-cmake/.

Reply:
Done at ec188143a4468e9f48da8a83ac2b4bfd1219e189.